test/python/sdpa/fp16.py

```
@@ -76,6 +78,10 @@ def validate_config(cfg):
     if cfg.is_ragged:
         assert cfg.is_padding == True, "is_ragged=True and is_padding=False not allowed"
 
+    if cfg.is_cu_seq_len:
+        assert cfg.is_padding == True, "is_cu_seq_len=True requires is_padding=True"
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `cfg.is_padding:` for truth checks

Replace with `cfg.is_padding`

(E712)